src/main/java/com/smartbear/readyapi/client/result/TestStepResult.java

```diff
@@ -50,6 +50,15 @@ public Long getTimeTaken() {
         return testStepResultReport.getAssertionStatus();
     }
 
+    public TestStepResultReport.AssertionStatusEnum getStatusForAssertion(String assertionName) {
+        for (String message : testStepResultReport.getMessages()) {
+            if (message.startsWith("[" + assertionName + "]")) {
```

Comment:
I will add the list of assertions to testStepResultReport, present on the test step, so that we can check that assertion was present on the test step. But that will be a separate PR.